2-mandatory/3-financial-times.js

```diff
@@ -23,15 +34,29 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    let arr=[]
+    for (let article of allArticleTitles){
+        for (let char of article){
+            if (char>="0" && char<="9"){
```

Comment:
cool!

Reply:
You could have also used regular expressions here:
https://stackoverflow.com/questions/5778020/check-whether-an-input-string-contains-a-number-in-javascript